events/firehose.go

```diff
@@ -25,6 +25,13 @@ const (
 	KinesisFirehoseTransformedStateProcessingFailed = "ProcessingFailed"
 )
 
+// Constants used for otf operation for the record
+const (
+	KinesisFirehoseOtfOperationInsert = "insert"
+	KinesisFirehoseOtfOperationUpdate = "update"
+	KinesisFirehoseOtfOperationDelete = "delete"
```

Comment:
Expected `type KinesisFirehoseOTFOperation string`, like is done elsewhere for similar enum-likes in other events